app/src/main/java/com/android/solvit/provider/ui/profile/ProfessionalProfile.kt

```diff
@@ -0,0 +1,324 @@
+            TitleText("Location", testTag = "locationTitle")
+            BodyText(provider.location.name, testTag = "location")
+          }
+
+          var isOpen by remember { mutableStateOf(true) }
+
+          Column {
+            TitleText("Position", testTag = "positionTitle")
+            Spacer(modifier = Modifier.width(8.dp))
+
+            Row(
+                // TODO : Stores the value position in memory
+                modifier = Modifier.testTag("position"),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically) {
+                  Box(
+                      modifier =
+                          Modifier.size(32.dp, 16.dp)
+                              .background(Color(0xFF4A4A4A), shape = CircleShape)
+                              .clickable { isOpen = !isOpen },
+                      contentAlignment =
+                          if (isOpen) Alignment.CenterEnd else Alignment.CenterStart) {
+                        Box(
+                            modifier =
+                                Modifier.size(12.dp)
+                                    .background(
+                                        if (isOpen) Color.Green else Color(239, 70, 55),
+                                        shape = CircleShape))
+                      }
+
+                  Spacer(modifier = Modifier.width(8.dp))
+
+                  Text(
+                      text = if (isOpen) "open" else "close",
+                      color = if (isOpen) Color.Green else Color(239, 70, 55),
+                      fontSize = 14.sp,
+                      fontWeight = FontWeight.Bold)
+                }
+          }
+        }
+  }
+}
+
+@Composable
+fun JobsDoneSection() {
+  // TODO : Change the hardcoded value to jobs done by the provider
+  Column(modifier = Modifier.fillMaxWidth().background(Color.White).padding(16.dp)) {
+    Text(
+        "Jobs done",
+        fontWeight = FontWeight.Bold,
+        fontSize = 18.sp,
+        modifier = Modifier.align(Alignment.CenterHorizontally).testTag("jobsDoneTitle"))
+    Spacer(modifier = Modifier.height(20.dp))
+    Row(horizontalArrangement = Arrangement.SpaceBetween, modifier = Modifier.fillMaxWidth()) {
+      JobItem("Back end")
+      JobItem("Front end")
+      JobItem("Visual Designer")
+      JobItem("Voyager")
+    }
+  }
+}
+
+@Composable
+fun JobItem(title: String) {
+  Box(
+      modifier =
+          Modifier.size(80.dp)
+              .shadow(4.dp, shape = RoundedCornerShape(12.dp))
+              .background(Color.White, shape = RoundedCornerShape(12.dp))
+              .padding(8.dp)
+              .testTag("jobItem"),
+      contentAlignment = Alignment.Center) {
+        Text(
+            text = title,
+            fontSize = 14.sp,
+            textAlign = TextAlign.Center,
+            maxLines = 2,
+            color = Color(239, 70, 55))
+      }
+}
+
+@Composable
+fun StatsSection(provider: Provider) {
```

Comment:
Nit: The columns are not aligned in the UI

Reply:
I know, it’s because some lines have 2 information and another 3. Anyway when we have all the information, I will update this screen

Reply:
I know, it’s because some lines have 2 information and another 3. Anyway when we have all the information, I will update this screen

Reply:
I was referring to the first 2 rows with both 2 elements. But I agree this is a nitpick and should be addressed when the UI is able to access all the necessary information